jmix-security/security-flowui/src/main/java/io/jmix/securityflowui/view/resourcerole/ResourceRoleModelDetailView.java

```diff
@@ -131,6 +133,15 @@ protected void initExistingEntity(String serializedEntityCode) {
         ResourceRole roleByCode = roleRepository.findRoleByCode(code);
 
         ResourceRoleModel resourceRoleModel = roleModelConverter.createResourceRoleModel(roleByCode);
+
+        // Sort resource policies in acs by default
+        resourceRoleModel.setResourcePolicies(
```

Comment:
Sort policies in `io.jmix.securityflowui.model.RoleModelConverter#createResourcePolicyModels` instead